### DISCUSSION TITLE
Can Sionna simulate the motion of an object?

### DISCUSSION THREAD

**Discussion by vcyycv0** [NVlabs/sionna#497] [Q&A]
I want to add a moving metal plate in the middle of Tx and rx. How do I add motion velocity and trajectory properties to the metal plate?

**Comment by SebastianCa** [comment]
Hi @vcyycv0,
Please take a look at the [Mobility Notebook](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Mobility.html).
